src/bcccp/carpark/exit/ExitController.java

```diff
@@ -13,7 +13,7 @@
 	
 	private enum STATE { IDLE, WAITING, PROCESSED, REJECTED, TAKEN, EXITING, EXITED, BLOCKED } 
 	
-	private STATE state;
+	public STATE state;
```

Comment:
STATE should remain private and have a public getter method

Reply:
Fixed.